src/System.Utf8String.Experimental/ref/System.Utf8String.Experimental.csproj

```diff
@@ -5,7 +5,7 @@
     <Configurations>netcoreapp-Debug;netcoreapp-Release</Configurations>
   </PropertyGroup>
   <ItemGroup Condition="'$(IsPrerelease)' != 'false'">
-    <Compile Include="System.Utf8String.cs" />
+    <Compile Include="System.Utf8String.Experimental.cs" />
```

Comment:
cc @GrabYourPitchforks - fyi regarding this rename

Reply:
Why would we be renaming the file?

Reply:
To be consistent with the pattern within the repo for all other projects. The ref cs file name equals the csproj file name (which equals the root directory name), I think.